gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/traversal-strategy.js

```diff
@@ -69,7 +70,32 @@ class TraversalStrategy {
   }
 }
 
+class VertexProgramStrategy extends TraversalStrategy {
+
+  constructor(graphComputer, workers, persist, result, vertices, edges, configuration) {
+    super();
+    this.configuration = new Map();
+    if (graphComputer !== undefined)
+      this.configuration.set("graphComputer", graphComputer);
+    if (workers !== undefined)
+      this.configuration.set("workers", workers);
+    if (persist !== undefined)
+      this.configuration.set("persist", persist);
+    if (result !== undefined)
+      this.configuration.set("result", result);
+    if (graphComputer !== undefined)
```

Comment:
Typo :)

Reply:
that was pretty useless of me :disappointed:  - fixed (hopefully)